api/node/__test__/js_value_conversion.spec.ts

```diff
@@ -248,57 +249,141 @@ test('get/set brush properties', (t) => {
 
   let black = instance!.getProperty("black");
 
-  t.is((black as Brush).toString(), "#000000ff");
+  t.is((black as SlintBrush).toString(), "#000000ff");
 
-  if (t.true((black instanceof Brush))) {
-    let blackColor = (black as Brush).color;
-    t.deepEqual(blackColor.red, 0);
-    t.deepEqual(blackColor.green, 0);
-    t.deepEqual(blackColor.blue, 0);
+  if (t.true((black instanceof SlintBrush))) {
+    let blackSlintRgbaColor = (black as SlintBrush).color;
+    t.deepEqual(blackSlintRgbaColor.red, 0);
+    t.deepEqual(blackSlintRgbaColor.green, 0);
+    t.deepEqual(blackSlintRgbaColor.blue, 0);
   }
 
   instance?.setProperty("black", "#ffffff");
   let white = instance!.getProperty("black");
 
-  if (t.true((white instanceof Brush))) {
-    let whiteColor = (white as Brush).color;
-    t.deepEqual(whiteColor.red, 255);
-    t.deepEqual(whiteColor.green, 255);
-    t.deepEqual(whiteColor.blue, 255);
+  if (t.true((white instanceof SlintBrush))) {
+    let whiteSlintRgbaColor = (white as SlintBrush).color;
+    t.deepEqual(whiteSlintRgbaColor.red, 255);
+    t.deepEqual(whiteSlintRgbaColor.green, 255);
+    t.deepEqual(whiteSlintRgbaColor.blue, 255);
   }
 
   let transparent = instance!.getProperty("trans");
 
-  if (t.true((black instanceof Brush))) {
-    t.assert((transparent as Brush).isTransparent);
+  if (t.true((black instanceof SlintBrush))) {
+    t.assert((transparent as SlintBrush).isTransparent);
   }
 
-  let ref = new Brush({ red: 100, green: 110, blue: 120, alpha: 255 });
+  let ref = new SlintBrush({ red: 100, green: 110, blue: 120, alpha: 255 });
   instance!.setProperty("ref", ref);
 
   let instance_ref = instance!.getProperty("ref");
 
-  if (t.true((instance_ref instanceof Brush))) {
-    let ref_color = (instance_ref as Brush).color;
+  if (t.true((instance_ref instanceof SlintBrush))) {
+    let ref_color = (instance_ref as SlintBrush).color;
     t.deepEqual(ref_color.red, 100);
     t.deepEqual(ref_color.green, 110);
     t.deepEqual(ref_color.blue, 120);
     t.deepEqual(ref_color.alpha, 255);
   }
 
+  instance!.setProperty("ref", { color: { red: 110, green: 120, blue: 125, alpha: 255 }});
```

Comment:
I Slint I can initialize a brush property just from a color. Maybe it wasn't clear in the API review, but I meant that IMO we should allow initialising brush properties in the JS API also from colors.

Or is there a particular reason to make the JS API behave different than what we can do in .slint files?

The intermediate "color" property is meant to be private (we should use a symbol anyway).

Reply:
Sorry my fault. Then it was a misunderstanding. You're right, it make totally sense!

Reply:
I think it should work both. With the interface `Brush` that can later extended e.g. for gradients and directly with a color.

Reply:
done.